testdata/templates/custom_actions.tpl

```diff
@@ -1,3 +1,107 @@
+{{- define "custom_action1_response" }}
```

Comment:
why are we duplicating these?  There were already copies of this data below.

Reply:
some needed tweaking and a third needed adding for actual pagination testing.  originals were preserved because when I modified them, things in the rest of the file broke :/

Reply:
will cut a new issue for that for sure.